operations/multi-tenancy/ExampleWorkflow.md

```diff
@@ -0,0 +1,755 @@
+# Example Workflow
+
+* [Overview](#overview)
+* [Configuration](#configuration)
+    * [Tenant Configuration](#tenant-configuration)
+        * [Tenant Names](#tenant-names)
+        * [Tenant API version](#tenant-api-version)
+    * [`Slurmcluster` Configuration](#slurmcluster-configuration)
+        * [`Slurmcluster` Names](#slurmcluster-names)
+        * [`Slurmcluster` IP Addresses](#slurmcluster-ip-addresses)
+        * [`Slurmcluster` API version](#slurmcluster-api-version)
+        * [`Slurmcluster` Configurable Values](#slurmcluster-configurable-values)
+        * [`Slurmcluster` Version Numbers](#slurmcluster-version-numbers)
+    * [`Slurmcluster` Slurm configuration](#slurmcluster-slurm-configuration)
+        * [`Slurmcluster` Slingshot VNI Allocation](#slurmcluster-slingshot-vni-allocation)
+        * [`Slurmcluster` Partitions and Nodes](#slurmcluster-partitions-and-nodes)
+    * [USS Configuration](#uss-configuration)
+* [Step-by-Step Guide](#step-by-step-guide)
+    * [Create tenant configuration file](#create-tenant-configuration-file)
+    * [Create `Slurmcluster` configuration file](#create-slurmcluster-configuration-file)
+    * [Edit Slurm configuration file](#edit-slurm-configuration-file)
+    * [Configure USS group variables](#configure-uss-group-variables)
+    * [Create BOS session template](#create-bos-session-template)
+    * [Create CFS configuration](#create-cfs-configuration)
+    * [Boot and Run](#boot-and-run)
+* [Status and Troubleshooting](#status-and-troubleshooting)
+    * [Tenant command examples](#tenant-command-examples)
+    * [`Slurmcluster` command examples](#slurmcluster-command-examples)
+    * [HSM command examples](#hsm-command-examples)
+    * [HNS command example](#hns-command-example)
+* [Appendices](#appendices)
+    * [Appendix A - `Noname` tenant](#appendix-a---noname-tenant)
+    * [Appendix B - `Noname Slurmcluster`](#appendix-b---noname-slurmcluster)
+    * [Appendix C - Slurm configuration](#appendix-c---slurm-configuration)
+    * [Appendix D - USS group variables](#appendix-d---uss-group-variables)
+
+## Overview
+
+A tenant is a collection of nodes that is dedicated to one particular set of users on an HPE Cray EX computer system running CSM.
+This guide is intended to provide a comprehensive set of instructions for a system administrator to configure, deploy, and run applications on, one or two tenants.
+
+In this document we provide examples for a hypothetical system called `Noname`, which has two tenants, and each tenant has a `Slurmcluster`.
+
+Please note that this document reflects the current state of the Multi-Tenancy feature.  For example, VNI blocks must be manually configured today, but they will be automatically configured in a future release.
+
+Here are the steps required:
+
+* Configure desired tenants and `Slurmclusters`
+* Deploy each tenant and its `Slurmcluster`
+* Configure each `Slurmcluster`'s `slurm.conf`
+* Configure the primary (`user` namespace) `slurm.conf`
```

Comment:
I now think this woud be better:  Make any required changes (e.g. VNIs) to primary (user namespace) slurm.conf

Reply:
updated